dsa/validate.go

```diff
@@ -1,51 +1,104 @@
 package dsa
 
 import (
+	"errors"
+
 	"github.com/prebid/prebid-server/v2/exchange/entities"
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/prebid/prebid-server/v2/util/jsonutil"
+)
+
+// ObjectSignal represents publisher DSA object required statuses
+type ObjectSignal int
```

Comment:
Nitpick: Enum types should be `int8` to match what they're compared against in the request.